Traceback (most recent call last):
  File "/Users/stepanzh/Venv/jupyterbook/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/stepanzh/Venv/jupyterbook/lib/python3.8/site-packages/nbclient/client.py", line 1117, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/stepanzh/Venv/jupyterbook/lib/python3.8/site-packages/nbclient/util.py", line 78, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/stepanzh/Venv/jupyterbook/lib/python3.8/site-packages/nbclient/util.py", line 57, in just_run
    return loop.run_until_complete(coro)
  File "/usr/local/opt/python@3.8/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/Users/stepanzh/Venv/jupyterbook/lib/python3.8/site-packages/nbclient/client.py", line 558, in async_execute
    await self.async_execute_cell(
  File "/Users/stepanzh/Venv/jupyterbook/lib/python3.8/site-packages/nbclient/client.py", line 862, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/Users/stepanzh/Venv/jupyterbook/lib/python3.8/site-packages/nbclient/client.py", line 765, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
L, U, p = lufact(A);
@show p;
------------------

BoundsError: attempt to access Tuple{LowerTriangular{Float64, Matrix{Float64}}, UpperTriangular{Float64, Matrix{Float64}}} at index [3]

Stacktrace:
 [1] indexed_iterate(t::Tuple{LowerTriangular{Float64, Matrix{Float64}}, UpperTriangular{Float64, Matrix{Float64}}}, i::Int64, state::Int64)
   @ Base ./tuple.jl:86
 [2] top-level scope
   @ In[19]:1
 [3] eval
   @ ./boot.jl:360 [inlined]
 [4] include_string(mapexpr::typeof(REPL.softscope), mod::Module, code::String, filename::String)
   @ Base ./loading.jl:1116
LoadError: BoundsError: attempt to access Tuple{LowerTriangular{Float64, Matrix{Float64}}, UpperTriangular{Float64, Matrix{Float64}}} at index [3]

